<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab07_PNR</class>
 <widget class="QMainWindow" name="Lab07_PNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab07_PNR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_PNR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_PNR">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEdit_PNR">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>311</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_PNR">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Add To Tail</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear_PNR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Queue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete_PNR">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Head</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ammount_PNR">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>470</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get Ammount</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_max_PNR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get Max Value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_min_PNR">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>540</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get Min Value</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_copyAndPasteElem_PNR">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>540</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Copy and
Paste element</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableSet_PNR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listSet_PNR">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_setFromTable_PNR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set from Table:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_setFromList_PNR">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set from List:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_table_PNR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Table:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_list_PNR">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>List:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_textEdit_PNR">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Text Edit:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_getFromTable_PNR">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>610</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get from Table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_getFromList_PNR">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>610</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get from List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mean_PNR">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>470</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Get Arithmetical  
Mean</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_value_PNR">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>430</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Value:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name_PNR">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>470</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name_PNR">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>430</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_first_PNR">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>540</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_second_PNR">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>540</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_copyAndPaste_PNR">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Copy and Paste:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_value_PNR">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>470</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_min_PNR">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>500</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_max_PNR">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>530</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mean_PNR">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>560</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mean:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_minVal_PNR">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>500</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_maxVal_PNR">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>530</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_meanVal_PNR">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>560</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setRowCount_PNR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Row
Count</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_row_PNR">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>600</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rowCount_PNR">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>570</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Row count:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_multByVal_PNR">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Multiply by
Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ammount_PNR">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>590</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ammount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ammountVal_PNR">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>590</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_isEmpty_PNR">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>540</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Is Empty?</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_isEmpty_PNR">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>620</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Is Empty:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_isEmptyVal_PNR">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>620</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setListItem_PNR">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>610</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add to List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setListItem_PNR_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>610</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remove from
List</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_rowInList_PNR">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>660</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rowCount_PNR_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>630</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Row in List:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
